src/objects.c

```diff
@@ -2484,7 +2479,7 @@ void func_80011264(ObjectModel *model, Object *obj) {
     batch = model->batches;
 
     while (i < model->numberOfBatches) {
-        if (batch[i].flags & 0x10000) {
+        if (batch[i].flags & OBJ_FLAGS_ANIMATED) {
```

Comment:
```suggestion
        if (batch[i].flags & BATCH_FLAGS_TEXTURE_ANIM) {
```

batchflag enum exists already